subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_sync.c

```diff
@@ -217,6 +217,14 @@ static int prepare_cb(struct lll_prepare_param *p)
 	remainder = p->remainder;
 	start_us = radio_tmr_start(1, ticks_at_start, remainder);
 
+#if defined(CONFIG_BT_CTLR_PROFILE_ISR) || \
+	defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
+	/* capture end of AUX_SYNC_IND/AUX_CHAIN_IND PDU, used for calculating
+	 * next PDU timestamp.
+	 */
+	radio_tmr_end_capture();
```

Comment:
Why the PDU end timer capture is enabled here? And why it is conditionally compilated?

Reply:
- In Periodic Advertising without chaining there was no use for LLL to get the end time from radio, hence there was no call to `radio_tmr_end_capture` to capture the radio end time
- With chaining the `sw_switch` used PPI/DPPI for back to back Tx, no radio end time capture was needed there either
- For PA LNA (and ISR profiling), the radio end time is required to setup the GPIOTE using `radio_gpio_pa_lna_enable` which needs call to `radio_tmr_tifs_base_get`, both PA/LNA and ISR profiling call `radio_tmr_end_get`

Reply:
Could you add this information to comment above the `radio_tmr_end_capture` call? This is something that helps a lot to understand why the call is there and why it is conditionally compiled.